src/tests/ftest/util/daos_utils.py

```diff
@@ -604,6 +604,25 @@ def container_list_attrs(self, pool, cont, sys_name=None, verbose=False):
             ("container", "list-attrs"), pool=pool, cont=cont, sys_name=sys_name,
             verbose=verbose)
 
+    def container_list_objects(self, pool, cont, sys_name=None):
+        """Call daos container list-objects.
+
+        Args:
+            pool (str): Pool UUID.
+            cont (str): Container UUID.
```

Comment:
Suggestion - commands accept UUID or labels
```suggestion
            pool (str): Pool UUID or label
            cont (str): Container UUID or label
```

Reply:
Done